ex1/test_x_mix_drix.py

```diff
@@ -1,18 +1,14 @@
 import unittest
 
-from ex1.exercise_1_x_mix_drix import XOBoard, PlayerToken, XOGame
+from exercise_1_x_mix_drix import XOBoard, PlayerToken, XOGame
 
 X = PlayerToken.X
 O = PlayerToken.O
```

Comment:
Add a similar short named variable for `PlayerToken.EMPTY`, and replace all occurrences in this file.